sys-libs/pam/pam-1.5.1.ebuild

```diff
@@ -51,11 +51,9 @@ multilib_src_configure() {
 	# Do not let user's BROWSER setting mess us up. #549684
 	unset BROWSER
 
-	# Disable automatic detection of libxcrypt; we _don't_ want the
-	# user to link libxcrypt in by default, since we won't track the
-	# dependency and allow to break PAM this way.
-
-	export ac_cv_header_xcrypt_h=no
+	if ! use xcrypt; then
```

Comment:
this is simply not enough, because:

a.) you must also make sure glibc is built without the crypt use flag
b.) you must pull libxcrypt if this flag is enabled

Reply:
Very good observations, thank you, I will add these changes.